### PR TITLE
Feat : Profile Page Created

### PR DESCRIPTION
**Pull request by gopalbbharadva**
### Features
-> It's responsive

Live Link: https://deploy-preview-4--sm-spotter.netlify.app/
P.s: Click on profile icon in navbar to check profile page.

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -18,12 +19,14 @@ export const Navbar = () => {
...
-              className="w-full h-full object-cover rounded-full "
-              src="https://treatiseui.netlify.app/Images/ian-dooley-lg.jpg"
-            />
-          </div>
+          <Link to="/profile">
+            <div className="w-8 h-8 cursor-pointer">
+              <img
+                className="w-full h-full object-cover rounded-full "
+                src="https://treatiseui.netlify.app/Images/ian-dooley-lg.jpg"
+              />
```
Comment (srishti-maurya) [inline src/components/Navbar/Navbar.jsx]: you may add the `alt` attribute to `img`

**Review by manojasarsa — APPROVED** [review]
Code looks good!

File: src/page/Profile/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <div
+          className="w-full max-w-xl m-auto border-b-2 flex justify-start
+          flex-col p-3 xs:flex-row 
+         "
+        >
+          <div className="p-2 w-32 h-32 flex xs:self-center">
+            <img
+              className="w-full h-full object-cover rounded-full"
+              src="https://treatiseui.netlify.app/Images/ian-dooley-lg.jpg"
+              alt=""
```
Comment (srishti-maurya) [inline src/page/Profile/Profile.jsx]: you may give `alt` a value

**Review by srishti-maurya — APPROVED** [review]
UI looks clean and elegant.
Code is readable and well managed with barrel exports.

File: src/page/Profile/Profile.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { IoSettingsOutline } from "react-icons/io5";
+import { PostCard } from "../../components/componentExport";
+
+export const Profile = () => {
+  return (
+    <div className="w-screen h-screen">
+      <div className="flex flex-col justify-center items-center">
+        <div
+          className="w-full max-w-xl m-auto border-b-2 flex justify-start
```
Comment (KoushikKintali) [inline src/page/Profile/Profile.jsx]: instead of writing multiple classNames, it would be better we can write in a single className

**Review by KoushikKintali — APPROVED** [review]

File: src/page/Profile/Profile.jsx (hunk 3 above)
Comment (gopalbbharadva) [reply src/page/Profile/Profile.jsx]: ya, it's good idea, I also feel to do so. thanks

**Pull request merged**